### PR TITLE
Make application runable by changing shortcode to avoid duplication

### PR DESCRIPTION
**Pull request by da-kami**

### CODE REVIEW

File: swap/src/cli.rs [hunk 1]
```diff
@@ -107,7 +107,7 @@ pub enum Command {
         // TODO: The listen address is only relevant for Alice, but should be role independent
         //  see: https://github.com/comit-network/xmr-btc-swap/issues/77
         #[structopt(
-            short = "a",
+            short = "la",
```
Comment (D4nte) [inline swap/src/cli.rs:110]:
```suggestion
            short = "pa",
```
`l` is not even present in the `long` version.

**Review by D4nte — CHANGES_REQUESTED** [review]
Please use later present in the long description.

**Review by rishflab — APPROVED** [review]

File: swap/src/cli.rs (hunk 1 above)
Comment (da-kami) [reply swap/src/cli.rs:110]: Crap, was looking at the variable name when creating the short-code 😅
Maybe we should just get rid of the short-codes altogether, they have caused nothing but trouble...

**Comment by da-kami** [comment]
@D4nte after thinking about this again: How about we get rid of all the shortcodes for now? They have cause nothing but trouble, I regret adding them when working on the CLI... 
They could easily be added later.

**Comment by da-kami** [comment]
Closing this in favor of https://github.com/comit-network/xmr-btc-swap/pull/108

**Pull request closed without merging**